tests/views/test_mau_views.py

```diff
@@ -30,6 +30,8 @@ class TestSiteMauLiveMetricsViewSet(BaseViewTest):
 
     @pytest.fixture(autouse=True)
     def setup(self, db, settings):
+        if organizations_support_sites():
+            settings.FEATURES['FIGURES_IS_MULTISITE'] = True
```

Comment:
```suggestion
        settings.FEATURES['FIGURES_IS_MULTISITE'] = organizations_support_sites()
```

Although this is practically the same, in unit tests `if` statements are not recommended.

Reply:
Good suggestion, thanks @OmarIthawi !